Form-Controls/index.html

```diff
@@ -10,18 +10,57 @@
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1 class="header">Amir T-Shirt Gallery</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>T-Shirt Order Form</title>
+   
+    </head>
+    <body class="form">
```

Comment:
question: why might this block not be necessary? Notice you already have an h1 above this